### ISSUE TITLE
Spelling fixes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8590]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)| @scop reported|
|-|-|
|Trac ID|trac#8590|
|Type|enhancement|
|Created|2016-07-12 10:58:44Z|
|Branch|https://github.com/twisted/twisted/tree/8590-scop-spelling|

<details><summary>Searchable metadata</summary>

```
trac-id__8590 8590
type__enhancement enhancement
reporter__scop scop
priority__low low
milestone__None None
branch__8590_scop_spelling 8590-scop-spelling
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1468321124085446 1468321124085446
changetime__1468332333443748 1468332333443748
version__None None
owner__scop scop

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)<a name="note_1"></a>|@scop commented|
|-|-|

https://github.com/twisted/twisted/pull/317

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban set owner to @scop|
|-|-|

Changes look good. Thanks!

I am waiting for the buildbot results and will merge.

What tool was used to generated these changes?

**Comment by twisted-trac** [comment]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)<a name="note_3"></a>|@scop commented|
|-|-|

Trac is rejecting my comments as spam, so I replied in the GH PR instead.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set status to `closed`|
|-|-|

In changeset cdf6441ea0b46b2b65c31be55c3f651f920ab9c2
```
#!CommitTicketReference repository="" revision="cdf6441ea0b46b2b65c31be55c3f651f920ab9c2"
Merge 8590-scop-spelling: Clean code of spelling errors.

Author: scop
Reviewer: Alex, adiroiban
Fixes: #8590
```
